Formula/zenity.rb

```diff
@@ -17,6 +18,8 @@ class Zenity < Formula
   depends_on "gtk+3"
   depends_on "gnome-doc-utils"
   depends_on "scrollkeeper"
+  depends_on "webkitgtk" => :recommended
```

Comment:
I'd _really_ like to avoid making this recommended if possible. It's a frankly massive dependency.